### ISSUE TITLE
[Feature Request]: Links and html in checkbox-lebel

### ISSUE THREAD

**Issue by vogelor** [tasteui/tasteui#129]
### Suggestion Theme

Sometimes it is necessary to add a link or something like this in the label of the checkbox.
At the moment this is not possible

### Main Benefits

For example if you want
[x] i accept the terms and conditions of this site 
with a link to the terms and conditions

### Notes

try this
```blade
        <x-checkbox>
            <x-slot:label>
                <b>Receive</b> Alert<br>
                xxx<br>
                xxx<br>
                xxx<br>
            </x-slot:label>
        </x-checkbox>
```
then you see what i mean...

**Comment by devajmeireles** [comment]
Hey, @vogelor .

This is a very very very good idea. We will add this possibility as soon as possible, maybe today or tomorrow. 👍🏻

**Comment by vogelor** [comment]
and please check label with multi lines.
if the checkbox has multible lines i believe the checkbox (rectangle) itself has to be at the TOP of the Text, NOT in the mittle.

OK:
```
[ ] very very
    very long 
    label
```

NOT OK:
```
    very very
[ ] very long 
    label
```

**Comment by devajmeireles** [comment]
Thanks, @vogelor .

This and #128 were addressed at #130 also with the addition of `xs` variation.

Due to our policy, I'm closing this issue. Feel free to open a new issue whenever necessary.
